containers/build.sh

```diff
@@ -3,13 +3,24 @@ set -eo pipefail
 
 image_name=$1
 org_name=$2
-platform=$3
+push=0
+if [[ $3 == "--push" ]]; then
+  push=1
+fi
+tag_suffix=$4
 
-echo "Building: $image_name for platform: $platform"
-tags=()
```

Comment:
Is this `tags=()` intentionally removed? Or it is a mistake?

Reply:
good catch!